onefile

```diff
@@ -0,0 +1 @@
+Something in a file
```

Comment:
I needed to create a file inside a folder to create a folder... it seems that I cannot (yet) move files using the github interface